Transaction.cs

```diff
@@ -0,0 +1,38 @@
+﻿public class Transaction
+{
+    private string date;
+    public string from;
+    public string to;
```

Comment:
public before private please 🙂  And can these be `readonly` or properties with a private `set` method? It's best not to have fully public variables if possible 🙂

Reply:
The variables can actually be completely private, I noticed they were public while working on part 2 but lost my train of thought after swapping branches to change it. I'll keep the readonly/private set in mind in the future.